### DISCUSSION TITLE
Sort by : Name

### DISCUSSION THREAD

**Discussion by echdareez** [BlafKing/sd-civitai-browser-plus#269] [Ideas]
The title says it all : would be awesome if there was an option to sort the results by name :-)
thanks!

**Comment by BlafKing** [comment]
Unfortunately this functionality would have to come from the CivitAI API, which is responsible for returning all models/search results.

If you'd like to have this functionality implemented, you can post a suggestion on the CivitAI github instead [here](https://github.com/civitai/civitai)!
